src/main/java/com/fatec/springapi4/entity/PartnerTrack.java

```diff
@@ -0,0 +1,37 @@
+package com.fatec.springapi4.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import java.time.LocalDate;
+
+@Getter @Setter @AllArgsConstructor
```

Comment:
Em todas as entidades, para que o JPA funcione corretamente, é necessário adicionar a anotação `@NoArgsConstructo` que seria a anotação para criação de um construtor sem argumentos.

quando a conexão estiver pronta e o projeto for executar, o JPA vai reclamar pois ele precisa de um construtor sem argumentos.

link de referência
[fórum alura](https://cursos.alura.com.br/forum/topico-allargsconstructor-e-noargsconstructor-qual-o-real-uso-269551) - "@NoArgsConstructor" Qual o real uso? 
[stackoverflow](https://stackoverflow.com/questions/34241718/lombok-builder-and-jpa-default-constructor) - Lombok @Builder and JPA Default constructor